components/dashboard/AssessmentCompleteDialog.js

```diff
@@ -27,14 +27,14 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const MAX_WIDTH = 'sm';
+const INTERCOM_PRODUCT_TOUR_ID = '107212';
 
 function AssessmentCompleteDialog({ show, onClose, onLogActivityButtonClick }) {
   const classes = useStyles();
 
   const handleClickOpen = () => {
     onClose();
-    // TODO: set the correct tourId
-    // window.Intercom('startTour', tourId);
+    window.Intercom('startTour', INTERCOM_PRODUCT_TOUR_ID);
```

Comment:
Double check that this accepts a string and not an int?

https://developers.intercom.com/installing-intercom/docs/intercom-javascript#section-intercomstarttour-tourid

Reply:
changed to int.